modules/awae/src/AWAE.f90

```diff
@@ -1258,6 +1265,88 @@ subroutine AWAE_Init( InitInp, u, p, x, xd, z, OtherState, y, m, Interval, InitO
 
 
 
+contains
+   subroutine CheckModAmb3Boundaries()
+      real(ReKi)                       :: Dxyz
+      real(ReKi)                       :: ff_lim(2)
+      real(ReKi)                       :: hr_lim(2)
+      real(ReKi),          parameter   :: GridTol = 1.0E-3  ! Tolerance from IfW for checking the high-res grid (Mod_AmbWind=3 only).
+      type(FlowFieldType), pointer     :: ff                ! alias to shorten notation to fullfield
+      type(WindFileDat),   pointer     :: wfi               ! alias to shorten notation to WindFileInfo
+      character(1024)                  :: tmpMsg
+
+      ff  => p%IfW(nt)%FlowField 
+      wfi => IfW_InitOut%WindFileInfo
+
+      tmpMsg = NewLine//NewLine//'Turbine '//trim(Num2LStr(nt))//' -- Mod_AmbWind=3 requires the InflowWind high-res grid '// &
```

Comment:
I believe you mean to say that the FAST.Farm high-res grid must be entirely contained within the InflowWind flow field.

Reply:
Yes.  That is much better wording.  Changed.